packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientHostApiImpl.java

```diff
@@ -59,6 +61,18 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
       flutterApi.onReceivedRequestError(this, view, request, error, reply -> {});
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
+    @Override
+    public boolean onRenderProcessGone(WebView view, RenderProcessGoneDetail detail) {
+      flutterApi.onRenderProcessGone(this, view, detail, reply -> {});
+      return returnValueForOnRenderProcessGone;
```

Comment:
The  `onRenderProcessGone` is handled by Flutter asynchronously. Does the app not crash if this returns true while it waits for Flutter to handle the callback?

Reply:
The app does not crash if this returns true. It indicates that we want to handle the crash ourselves. In our case that would mean report error to Sentry and reload the webview.